src/qunit.cpp

```diff
@@ -953,9 +957,9 @@ void QUnit::TransformInvert(const complex& topRight, const complex& bottomLeft,
         [&]() { bare; });
 
 #define CTRLED_PHASE_WRAP(ctrld, ctrldgen, bare, anti)                                                                 \
-    ApplyEitherControlled(lcontrols, controlLen, { ltarget }, anti,                                                    \
+    ApplyEitherControlled(controls, controlLen, { target }, anti,                                                      \
         [&](QInterfacePtr unit, std::vector<bitLenInt> mappedControls) {                                               \
-            if (!shards[ltarget].isPlusMinus) {                                                                        \
+            if (!shards[target].isPlusMinus) {                                                                         \
```

Comment:
Trailing whitespace scattered throughout here.

Reply:
I will `make format`, apologies.

Reply:
Oh, these are my stupid macros, if you remember. I promise you I will refactor these entirely out, but let's not tackle that in this PR.

Reply:
(The historical reason for these was that you noticed the patterns were repetitive and asked me to refactor, but the method pointer typing is super obnoxious.)

Reply:
(The methods would also take 1 bajillion irregular parameters, without some significant investment in lambda fanciness.)

Reply:
Were these comments related to a different element?  Does trailing whitespace map to a macro in some form?

Reply:
Benn, yes. Sorry, what I mean is that this is not trailing white space; it's white space in multiple lines of a function-like macro. I was saying, I will ultimately remove these, but then went in months ago, and they're a little tangled and not in the scope of this PR. (I already removed one, in this pull.)

Reply:
Oh, man, my viewport wasn't quite wide enough to capture that this was in a macro.  I wonder why it syntax highlighted here at all.  Are you mixing whitespace and tabs?